package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "webpack-retry-chunk-load-plugin",
-  "version": "2.0.2",
+  "version": "2.0.3",
```

Comment:
Can you revert this please? It will get bumped automatically when doing `npm version patch` (which also adds the git tag)

Reply:
done!